optics-core/src/Optics/Generic.hs

```diff
@@ -82,7 +83,11 @@ data Void0
 --
 -- >>> user ^. gfield @"salary"
 -- ...
--- ...User’ doesn't have a field named ‘salary’
+#if __GLASGOW_HASKELL__ == 802
```

Comment:
this actually may disable doctests all together (because an empty line may be introduced by cpphs, so we have just `...` which matches everything).

Reply:
... `cabal haddock` should tell that. (it might also depend on `cpp` used).

Reply:
I solved it properly (and improved error messages of generic optics in the process).